examples/regression/2-advanced-analysis/plot_both_uncertainties.py

```diff
@@ -86,20 +89,52 @@ def get_1d_data_with_normal_distrib(
         ("linear", LinearRegression()),
     ]
 )
+polyn_model_quant = Pipeline(
+    [
+        ("poly", PolynomialFeatures(degree=degree_polyn)),
+        ("linear", QuantileRegressor(
+            alpha=1e-9,
+            fit_intercept=False,
```

Comment:
On personal experience, it would seem that `fit_intercept=False` seems to give better results, however, on the `QuantileRegressor` class it is set by default to `True`. So I will just keep it at that for the moment, simply using `QuantileRegressor` without setting `fit_intercept` to any value.